formatTest/unit_tests/expected_output/basicStructures.re

```diff
@@ -146,7 +146,7 @@ let x = arr^[0] = 1;
  *============================================================================
  */;
 
-let (/\/) = (+); /* // is not a comment */
+let (/++) = (+); /* // is not a comment */
```

Comment:
Forgive me I'm pretty new to Reason but, I think the comment about `// is not a comment` should be updated?

Reply:
Thanks! Updated.